File: src/main/java/org/jukeboxmc/player/info/UIProfile.java
package org.jukeboxmc.player.info;

import org.jetbrains.annotations.Nullable;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public enum UIProfile {

    CLASSIC,
    POCKET;

    public static @Nullable UIProfile getById(int id ) {
        return switch ( id ) {
            case 0 -> CLASSIC;
            case 1 -> POCKET;
            default -> null;
        };
    }

}

File: JukeboxMC-API/src/main/kotlin/org/jukeboxmc/api/player/info/UIProfile.kt
package org.jukeboxmc.api.player.info

/**
 * The ui profile that can be defined in the client's game settings
 */
enum class UIProfile {

    CLASSIC,
    POCKET;

    companion object {
        fun getById(id: Int): UIProfile? {
            return when (id) {
                0 -> CLASSIC
                1 -> POCKET
                else -> null
            }
        }
    }
}